department-of-veterans-affairs/caseflow
# Added paper trail versioning when changed_request_type updated

Resolves #15154 

### Description
Please explain the changes you made here.

### Acceptance Criteria

### Testing Plan
1. Open a rails console `make c`
1. Grab the last legacy appeal `appeal = LegacyAppeal.last`
1. Ensure the current version length is 1 `appeal.versions.length`
1. Update the request type `appeal.update(changed_request_type: "V")`
1. Check the version length is now 2 `appeal.versions.length`
1. Update a different field `appeal.update(home_loan_guaranty: true)`
1. Check the version length is still 2 `appeal.versions.length`
